modules/analysis-runner/src/cljdoc/analysis/runner.clj

```diff
@@ -20,8 +20,9 @@
            (java.net URI)
            (java.nio.file Files)))
 
-(defn copy [uri file]
-  (with-open [in  (io/input-stream uri)
+(defn copy [source file]
```

Comment:
Maybe this would be a reasonable thing to put into `cljdoc.util` since it's used in multiple places now.

Reply:
done